### PR TITLE
wip - specific query creation and deletion

### PR DESCRIPTION
**Pull request by connectbo**

### CODE REVIEW

**Comment by trel** [comment]
looking good.   thanks for the confirmation button on deletion and creation.

- top filter css is weird - display isn't quite right
- creation puts the values in the wrong column (alias and sqlstring are switched)
- new specific query does not honor the 'return' key to save
- needs column sorting

File: src/controllers/SpecificQueryController.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        url: `${restApiLocation}/admin`,
+        headers: {
+            'Accept': 'application/json',
+            'Authorization': localStorage.getItem('zmt-token')
+        },
+        params: {
+            action: 'add',
+            target: 'specificQuery',
+            arg2: alias,
+            arg3: sqlStr
```
Comment (trel) [inline src/controllers/SpecificQueryController.js:15]: these two are flipped

Comment (connectbo) [reply src/controllers/SpecificQueryController.js:15]: ahhh

**Comment by connectbo** [comment]
yep, the top filtering is not done yet.. I will refine its layout after the logic is set up, and then add sorting, keyevent.

**Comment by connectbo** [comment]
Added sql formatting and disabled delete button for 'ls'. sqlStr Textarea will auto-size itself, so it will also work on smaller screens. Ready for your review. Let me know what you think.

**Comment by trel** [comment]
alias column gets squeezed on window resize, probably should be fixed to width of longest alias string.

rest looks really good.

**Comment by connectbo** [comment]
Squashed these commits, will add # if it looks good.

**Comment by trel** [comment]
perfect - please # it.

**Comment by connectbo** [comment]
cool, added #.

**Pull request merged**